### DISCUSSION TITLE
Any best practices how to model VXLAN in netbox

### DISCUSSION THREAD

**Discussion by MalfuncEddie** [netbox-community/netbox#6499] [Q&A] [answered]
Any best practices how to model VXLAN in netbox? Is it possible of could I use custom fields to model this?

**Comment by ryanmerolle** [comment, marked as answer]
You can create custom fields to apply to a vlan group or vlan.  For example maybe a VNI.

You can create VLANs and assign to a site or VLAN Groups and assign to a rack, region, site, site group, cluster, or cluster group.

You can create prefix/vlan roles and tag it as VXLAN.

It all depends on your logic, but that's how I handle VXLAN currently.
